products/BellHybrid/apps/application-bell-focus-timer/data/FocusContract.hpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2017-2024, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+
+#include <apps-common/BasePresenter.hpp>
+#include <widgets/ListItemProvider.hpp>
+#include <apps-common/widgets/TimerWithCallbacks.hpp>
+#include <time/time_locale.hpp>
+
+namespace app::focus::contract
+{
+    class View
+    {
+      public:
+        virtual ~View() noexcept = default;
+    };
+
+    class SettingsPresenter : public BasePresenter<View>
+    {
+      public:
+        virtual ~SettingsPresenter() noexcept                                           = default;
+        virtual auto getPagesProvider() const -> std::shared_ptr<gui::ListItemProvider> = 0;
```

Comment:
Consider using `auto fun() -> type` for all declarations or 
```suggestion
        virtual std::shared_ptr<gui::ListItemProvider> getPagesProvider() const  = 0;
```
to be consistent in this file.